cli/src/main/kotlin/uk/gov/justice/digital/cli/editor/DomainEditor.kt

```diff
@@ -301,6 +302,8 @@ class DomainEditor(private val session: InteractiveSession, private val service:
                        viewText = pageElements[20].multiLineFieldValue()
                    ),
                    mapping = null, // Mapping is not currently supported
+                   tags = null, // Tags not currently supported
```

Comment:
Would it be better here to set it to an empty collection to tags and violations instead of null?

Reply:
Perhaps, but amend will take care of later if tags are subsequently provided.